README.md

```diff
@@ -20,7 +20,10 @@
 ## Team members
 
 ### realfranser
+### ManuelHuesoBejar
 
 * Name: Francisco Javier Serrano
 * Fun fact: I am a crypto bro like @PabloMartinMorales
 
```

Comment:
Move the name into this line, this should be the format:

### username
* Name
* Fun fact